src/crate/client/connection.py

```diff
@@ -105,6 +116,7 @@ def __exit__(self, *excs):
 def connect(servers=None,
             timeout=None,
             backoff_factor=0,
+            poolsize=10,
```

Comment:
Same here, please move to the last position.

Reply:
Thanks again!